### PR TITLE
[1.1] Load vars on PSERVER

### PR DESCRIPTION
**Pull request by seiriosPlus**

### CODE REVIEW

**Comment by jacquesqiao** [comment]
unit test？

**Review by typhoonzero — COMMENTED** [review]
Add tests of doing real save and load?

**Comment by seiriosPlus** [comment]
Add UT done, @jacquesqiao @typhoonzero

File: python/paddle/fluid/tests/unittests/test_dist_save_load.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        required_envs.update(need_envs)
+
+        if check_error_log:
+            required_envs["GLOG_v"] = "7"
+            required_envs["GLOG_logtostderr"] = "1"
+
+        model_dir = tempfile.mkdtemp()
+
+        local_env = {}
+        local_env["SAVE"] = "1"
```
Comment (typhoonzero) [inline python/paddle/fluid/tests/unittests/test_dist_save_load.py:53]: Should have case that dist save and dist load?

Comment (seiriosPlus) [reply python/paddle/fluid/tests/unittests/test_dist_save_load.py:53]: The UT is local save and dist load.
I will add the case that dist save and dist load

**Review by jacquesqiao — APPROVED** [review]
LGTM!

**Pull request merged**